packages/storybook/.storybook/preview.js

```diff
@@ -0,0 +1,48 @@
+import { withThemeByClassName } from "@storybook/addon-styling"
+import { action } from "@storybook/addon-actions"
+
+// Gatsby's Link overrides:
+// Gatsby Link calls the `enqueue` & `hovering` methods on the global variable ___loader.
+// This global object isn't set in storybook context, requiring you to override it to empty functions (no-op),
+// so Gatsby Link doesn't throw errors.
+global.___loader = {
+  enqueue: () => {},
+  hovering: () => {},
+}
+// This global variable prevents the "__BASE_PATH__ is not defined" error inside Storybook.
+global.__BASE_PATH__ = "/"
+
+// Navigating through a gatsby app using gatsby-link or any other gatsby component will use the `___navigate` method.
+// In Storybook, it makes more sense to log an action than doing an actual navigate. Check out the actions addon docs for more info: https://storybook.js.org/docs/react/essentials/actions
+
+window.___navigate = (pathname) => {
+  action("NavigateTo:")(pathname)
+}
+
+import "@thepolicylab-projectportals/gatsby-theme-project-portal/src/styles/global.css"
+/** @type { import('@storybook/react').Preview } */
+const preview = {
+  parameters: {
+    actions: { argTypesRegex: "^on[A-Z].*" },
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /(Date|opportunityCloses|created|lastModified)$/,
+      },
+    },
+  },
+
+  decorators: [
+    // Adds theme switching support.
+    // NOTE: requires setting "darkMode" to "class" in your tailwind config
```

Comment:
should we update our tailwind config in this pr?

Reply:
Ooh, I'd not even seen that. It didn't work at the first attempt, but we might be able to get it working. I think that will be a new issue.